mitodl/bootcamp-ecommerce
# Remove facebook pixel code, add google tag manager code

#### What are the relevant tickets?
- Closes #196 
- Closes #197 

#### What's this PR do?
- Removes Facebook pixel tracking
- Adds Google Tag Manager JS code

#### How should this be manually tested?
- Create a normal Google Analytics account.  
  - Set `GA_TRACKING_ID` to the correct value in your `.env` file (`UA-XXXXXXXXX-X`)
- Create a Google Tag Manager account.
  - Create a Universal Analytics tag with a tracking ID=`UA-999999999-9`, with PageView as a trigger.
  - Submit and publish the tag.
  - Set `GTM_TRACKING_ID` to the id of your Tag Manager account (`GTM-XXXXXXX`).
- Go to the bootcamp `/pay` (or login) page with the network tab open.  Verify that the following requests are sent:
  - `https://www.googletagmanager.com/gtm.js?id=GTM-XXXXXXX`
  - `https://www.google-analytics.com/collect` with parameter `t=pageview` and `tid=<your normal Analytics account>`.
  - `https://www.google-analytics.com/collect` with parameter `t=pageview` and `tid=UA-999999999-9`.
